File: Condicionales/Ejercicio2-Condicionales.cpp
/*2. Escriba un programa que lea tres números y determine cuál de ellos es el mayor.*/

#include<iostream>
using namespace std;

int main(){
    int num1, num2, num3;
    cout<<"Digite 3 numeros separados cada uno por un espacio: ";
    cin>>num1>>num2>>num3;

    if(num1==num2 && num1==num3){
        cout<<"Los numeros son iguales";
    }
    else if (num1==num2 && num2>num3){
        cout<<"El mayor es: "<<num1;
    }
    else if (num1==num2 && num2<num3){
        cout<<"El mayor es: "<<num3;
    }
    
    else if (num1>num2 && num1>num3){
        cout<<"El mayor es: "<<num1;
    }
    else if (num2>num1 && num2>num3){
        cout<<"El mayor es: "<<num2;
    }
    else{
        cout<<"El mayor es: "<<num3;
    }
    return 0;
}

File: Comparativa Python/Ejercicio2-Condicionales.py
# 2. Escriba un programa que lea tres números y determine cuál de ellos es el
# mayor.

n1 = int(input('Digite número 1: '))
n2 = int(input('Digite número 2: '))
n3 = int(input('Digite número 3: '))

if (n1 == n2 and n2 == n3):
    print('Números iguales')

elif (n1 >= n2 and n1 >= n3):
    print('El número mayor es: ', n1)

elif (n2 >= n1 and n2 >= n3):
    print('El número mayor es: ', n2)

else:
    print('El número mayor es: ', n3)
